lib/engines/hckinstall/iso.json

```diff
@@ -9,5 +9,16 @@
       "windows_image_names": "Windows Server 2019 SERVERSTANDARDCORE",
       "product_key": "AAAAA-AAAAA-AAAAA-AAAAA-AAAAA"
     }
+  }, 
+  "Win2022x64_GUI": {
+    "path": "en_windows_server_2022_x64_dvd_7f6f8e3e.iso",
+    "studio": {
+      "windows_image_names": "Windows Server 2022 SERVERSTANDARD",
+      "product_key": "BBBBB-BBBBB-BBBBB-BBBBB-BBBBB"
+    },
+    "client": {
+      "windows_image_names": "Windows Server 2022 SERVERSTANDARDCORE",
```

Comment:
It has `CORE` in the image name though the key is suffixed with `_GUI` instead of `_CORE`. Are you sure this is right?

Reply:
It's RFC version, i'll fix it, also we need to deciede what to do with our platform json's